versioned_docs/version-2.6/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -176,14 +174,24 @@ You will need to also manually enter the Graph, Token, and Auth Endpoints.
 
 From the Rancher UI, enter information about your AD instance hosted in Azure to complete configuration.
 
-Enter the values that you copied to your [text file](#tip).
-
 1. Log into Rancher.
+
 1.	In the top left corner, click **☰ > Users & Authentication**.
+
 1. In the left navigation menu, click **Auth Provider**.
+
 1. Click **AzureAD**.
+
 1. Complete the **Configure Azure AD Account** form using the information you copied while completing [Copy Azure Application Data](#4-copy-azure-application-data).
 
+    :::caution 
+    
+    The Azure AD user credentials entered in this step will be mapped to the Rancher local principal account and assigned administrator privileges. Make sure you are comfortable giving administrator privileges to this Azure AD account.
```

Comment:
```suggestion
    The Azure AD account will be granted administrator privileges, since its details will be mapped to the Rancher local principal account. Make sure that this level of privilege is appropriate before you continue.
```